website/docs/UserEvent.md

```diff
@@ -178,3 +178,55 @@ The `textInput` event is sent only for mutliline text inputs.
 **Leaving the element**:
 - `endEditing`
 - `blur`
+
+## `scroll()`
+
+```ts
+type(
+  element: ReactTestInstance,
+  options?: {
+    y?: number | number[],
+    x?: number | number[],
+    momentumY?: number | number[],
+    momentumX?: number | number[],
+  }
+```
+
+Example
+```ts
+const user = userEvent.setup();
+await user.scrollTo(scrollView, { y: 100, momentumY: 200 });
+```
+
+This helper simulates user scrolling a host `ScrollView` element. 
+
+This function supports only host `ScrollView` elements, passing other element types will result in error. Note that `FlatList` is accepted as it renders to a host `ScrolLView` element, however in the current iteration we focus on base `ScrollView` only.
+
+Scroll interaction should match host `ScrollView` direction. For vertical scroll view (`horizontal={false}`) you should pass only `y` (or `momentumY`) options, for horizontal scroll view (`horizontal={true}`) you should pass only `x` (or `momentumX`) options.
+
+Each scroll interaction consists of mandatory drag scroll part which simulates user dragging the scroll view with his finger (`y` or `x` option). This may optinally be followed by momentum scroll movement which simulates the inertial movement of scroll view content after user lifts his finger up (`momentumY` or `momentumX` options).
```

Comment:
Maybe it could be worth mentioning that we don't have access to content size and so props that rely on it like ` onViewableItemsChanged` or `onEndReached` will never be called?

Reply:
That being said the sequence of events is detailed at the end so it may not be necessary

Reply:
@pierrezimmermannbam good idea to mention content size thing as the current state of things. In the future we could actually work on improving the FlatList features, e.g. by allowing user to pass mock content size props or do some reasonable defaults, etc. I think it might be a topic worth exploring.